File: java/SwappingUsingXOR.java
public class Algorithms {    
    public static void main(String[] args) { 
        int a = 2002, b = 2001;
        

        //Swapping 
        a^=b;
        b^=a;
        a^=b;

        System.out.println("After Swapping : " +a+","+b); 

    } 
} 

File: c++/SwappingUsingXOR.cpp
#include<iostream>
using namespace std;

int main(){

    int a = 5, b = 3;

    

    //Swaping
    a ^= b;
    b ^= a;
    a ^= b;

    cout<<"After Swap :"<<a<<" , "<<b;

    return 0;

}